### PR TITLE
Only print loyalty when relevant

### LINKED ISSUE #256: All cards have <loyalty> value
The `loyalty` value is present for all cards, not only for planeswalkers.
It should be the same as with creatures, only they have a `pt` value.

First 5 cards from todays spoiler file:
```
  <cards>
    <card>
      <name>Chalice of the Void</name>
      <set rarity="Mythic Rare" picURL="https://c1.scryfall.com/file/scryfall-cards/normal/front/2/5/25674e46-e15b-4fc0-9813-39b4e1c23de4.jpg">TSR</set>
      <manacost>XX</manacost>
      <cmc>0.0</cmc>
      <type>Artifact</type>
      <loyalty>None</loyalty>
      <tablerow>1</tablerow>
      <text>Chalice of the Void enters the battlefield with X charge counters on it.
Whenever a player casts a spell with converted mana cost equal to the number of charge counters on Chalice of the Void, counter that spell.</text>
    </card>
    <card>
      <name>Path to Exile</name>
      <set rarity="Mythic Rare" picURL="https://c1.scryfall.com/file/scryfall-cards/normal/front/2/e/2ee3e42e-120c-468a-988d-4441b6025bbe.jpg">TSR</set>
      <manacost>W</manacost>
      <cmc>1.0</cmc>
      <color>W</color>
      <type>Instant</type>
      <loyalty>None</loyalty>
      <tablerow>3</tablerow>
      <text>Exile target creature. Its controller may search their library for a basic land card, put that card onto the battlefield tapped, then shuffle their library.</text>
    </card>
    <card>
      <name>Relentless Rats</name>
      <set rarity="Mythic Rare" picURL="https://c1.scryfall.com/file/scryfall-cards/normal/front/2/8/28de0281-0370-44df-9ee0-6e1bb9925a92.jpg">TSR</set>
      <manacost>1BB</manacost>
      <cmc>3.0</cmc>
      <color>B</color>
      <type>Creature - Rat</type>
      <pt>2/2</pt>
      <loyalty>None</loyalty>
      <tablerow>2</tablerow>
      <text>Relentless Rats gets +1/+1 for each other creature on the battlefield named Relentless Rats.
A deck can have any number of cards named Relentless Rats.</text>
    </card>
    <card>
      <name>Bane of Progress</name>
      <set rarity="Rare" picURL="https://c1.scryfall.com/file/scryfall-cards/normal/front/3/e/3ea495ff-33df-406b-98e3-5579e2158bbc.jpg">CC1</set>
      <manacost>4GG</manacost>
      <cmc>6.0</cmc>
      <color>G</color>
      <type>Creature - Elemental</type>
      <pt>2/2</pt>
      <loyalty>None</loyalty>
      <tablerow>2</tablerow>
      <text>When Bane of Progress enters the battlefield, destroy all artifacts and enchantments. Put a +1/+1 counter on Bane of Progress for each permanent destroyed this way.</text>
    </card>
    <card>
      <name>Command Tower</name>
      <set rarity="Rare" picURL="https://c1.scryfall.com/file/scryfall-cards/normal/front/3/b/3beb6fa5-44db-4056-b8ec-42547aa5581f.jpg">CC1</set>
      <manacost/>
      <cmc>0.0</cmc>
      <type>Land</type>
      <loyalty>None</loyalty>
      <tablerow>0</tablerow>
      <text>{T}: Add one mana of any color in your commander's color identity.</text>
    </card>
```

### PR DESCRIPTION
**Pull request by tooomm**
Fixes #256

Stop writing `<loyalty>None</loyalty>` values to the spoiler file.

Please double check this. Results are looking fine to me.

### CODE REVIEW

**Comment by tooomm** [comment]
With my recent PR being merged, you don't need everything setup and working locally to check changes... you can quickly review the [resulting output of every PR (appended as artifact)](https://github.com/Cockatrice/Magic-Spoiler/actions/runs/1052523934) next to the code now.

Reviewing PR's just got easier. 😉 

Only the [5 AFR planeswalkers](https://scryfall.com/search?q=e%3Aafr+t%3Aplaneswalker) hold the `<loyalty>` key in the `AFR.xml` file now. The file is considerable smaller as well as there are only 10 cards with that key now (all planeswalkers are contained 2 times, as there are two art variants each: https://github.com/Cockatrice/Magic-Spoiler/issues/269). Resulting in 391 lines less. (Compare to https://github.com/Cockatrice/Magic-Spoiler/blob/6dbeb984ac138fb969c8c523124704eb9753adac/AFR.xml created around the same time)

File: magic_spoiler/__main__.py [hunk 1]
```diff
@@ -373,7 +373,7 @@ def write_cards(
         if pow_tough:
             card_xml_file.write("<pt>" + pow_tough + "</pt>\n")
 
-        if "loyalty" in card.keys():
+        if str(card.get("loyalty")) != "None":
```
Comment (ebbit1q) [inline magic_spoiler/__main__.py:376]: this is bad python code

Comment (tooomm) [reply magic_spoiler/__main__.py:376]: That is very much possible. :)

Thank you for appending and improving it in #255!

Comment (ebbit1q) [reply magic_spoiler/__main__.py:376]: you're welcome

**Pull request closed without merging**